src/Compilers/Test/Core/Assert/AssertEx.cs

```diff
@@ -955,5 +955,22 @@ public static void NotNull<T>([NotNull] T value)
         }
 
 #nullable disable
+
+        public static void Throws<TException>(string expectedMessage, Action testCode)
+            where TException : Exception
+        {
+            TException e = null;
+            try
+            {
+                testCode();
+            }
+            catch (TException x)
+            {
+                e = x;
+            }
+
+            Assert.True(e != null, $"Expected {typeof(TException).Name} but no exception thrown.");
+            Assert.Equal(expectedMessage, e.Message);
+        }
```

Comment:
Why not just using the xunit helpers?

Reply:
I did not find a helper that validates message. Did I miss anything?